server/sonar-db-dao/src/test/java/org/sonar/db/purge/PurgeDaoTest.java

```diff
@@ -132,7 +134,7 @@ public void should_purge_inactive_short_living_branches() {
   @Test
   public void shouldDeleteHistoricalDataOfDirectoriesAndFiles() {
     dbTester.prepareDbUnit(getClass(), "shouldDeleteHistoricalDataOfDirectoriesAndFiles.xml");
-    PurgeConfiguration conf = new PurgeConfiguration(new IdUuidPair(THE_PROJECT_ID, "ABCD"), new String[] {Scopes.DIRECTORY, Scopes.FILE},
+    PurgeConfiguration conf = new PurgeConfiguration(new IdUuidPair(THE_PROJECT_ID, "ABCD"), asList(Qualifiers.DIRECTORY, Scopes.FILE),
```

Comment:
Bad usage of Scopes instead of Qualifiers. However this test does not verify that configuration is correct. The test provides its own list of qualifiers to be purged.